.mailmap

```diff
@@ -0,0 +1,16 @@
+Andrea Maria Piana <cammellos@cammelloss-MacBook-Pro.local>
```

Comment:
eemh, not sure this is meant to be here?

Reply:
why not? :)

Reply:
@cammellos This was just to make our names uniform across the repo. In this case there were commits that you've made to the repo using that email address. Do you prefer not to have that?

Reply:
Then maybe we should use status emails?

Reply:
didn'd even know what a mailmap file was :D , goddamn spelling errors on hostname!